### PR TITLE
update njks files to use macros where possible

### PR DESCRIPTION
**Pull request by robertparkinson**
Updated to use radioGroups where possible so we are consistent across pages.

### CODE REVIEW

File: src/main/app/forms/models/generalDamages.ts [hunk 1]
```diff
@@ -1,16 +1,14 @@
 export class GeneralDamages {
-  static readonly LESS = new GeneralDamages('LESS', 'less')
```
Comment (timja-kainos) [inline src/main/app/forms/models/generalDamages.ts]: Is display value needed still?

Comment (robertparkinson) [inline src/main/app/forms/models/generalDamages.ts]: missed one! thanks

File: src/main/app/forms/models/defendantAddition.ts [hunk 2]
```diff
@@ -10,9 +10,9 @@ export class DefendantAddition implements Serializable<DefendantAddition> {
 
   @IsDefined({ message: ValidationErrors.DEFENDANT_ADDITION_REQUIRED })
   @IsEnum(YesNo, { message: ValidationErrors.DEFENDANT_ADDITION_REQUIRED })
-  isAddDefendant?: YesNo
+  isAddDefendant?: string
```
Comment (rkondratowicz) [inline src/main/app/forms/models/defendantAddition.ts]: Shouldn't the `@IsEnum` annotation be removed/changed now?

**Comment by rkondratowicz** [comment]
Half of the changes on this pull request don't seem to be related to nunjucks macros.
Can it be split into separate ones?

**Pull request closed without merging**